tasks/main.yml

```diff
@@ -10,51 +10,32 @@
   stat: path=/usr/bin/gitlab-ctl
   register: gitlab_file
 
-# Install GitLab and its dependencies (RedHat).
-- name: Install GitLab dependencies (RedHat).
-  yum: "name={{ item }} state=installed"
+# Install GitLab and its dependencies.
+- name: Install GitLab dependencies.
```

Comment:
Can you still leave the "(RedHat)" in the name here? Helpful for those who are running on Debian and wondering why this task would be skipped...